Contact and animations modifications

odufour7 opened the pull request:
- I addressed your requests by adding docstrings, adjusting whitespaces, and updating the requirements.
- I modified the contact tab to directly interact with the CSV contact file sourced from the MADRAS My Core Cloud.
- I made some minor improvements to the animation style. Additionally, with the trajectories from the cloud, you can now view pedestrians from top-view cameras. 
- I completed the CCTV analysis in the tab analysis / density profile

Commit:
correction of animation_tab / contacts_tab / map_tab

Commit:
correction of animation_tab / contacts_tab / map_tab

Commit:
update contacts_tab and animation_tab

Commit:
update contacts_tab (overname) and animation (style)

Commit:
add analysis_CCTV

Commit:
add CCTV analysis in analysis tab density profile

Commit:
add CCTV analysis in analysis tab density profile

Commit:
update requirements and micro corrections CCTV analysis file

Commit:
improve contact tab CCTV analysis and animation tab

chraibi reviewed (changes requested):
Thanks Oscar, this looks nice!
Could you please rename other_datasets to something meaninful like for example "gps_trajectories".

Could you also fix the CI?

odufour7 commented:
"Could you also fix the CI?" What does it mean "CI" ?

Commit:
add white background

Commit:
add white background

odufour7 commented:
I added a white background to enhance readability when the web browser has a black background. And, I incorporated survey data and renamed the folders within the data directory

Commit:
survey_correction

Commit:
survey_correction

chraibi committed:
Merge branch 'main' into main
